api-configuration/src/main/java/org/jvalue/ods/apiConfigurationService/endpoint/ApiConfigurationService.java

```diff
@@ -0,0 +1,143 @@
+package org.jvalue.ods.apiConfigurationService.endpoint;
+
+import org.jvalue.ods.apiConfigurationService.helper.GraphqlQueryFactory;
+import org.jvalue.ods.apiConfigurationService.helper.SqlFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.Arrays;
+
+import static org.jvalue.ods.apiConfigurationService.helper.GraphqlQueryFactory.createSQLGraphQuery;
+
+@RestController
```

Comment:
You are using REST here as RPC names, which is typically a bad style. Instead we will want to use the HTTP verbs to address resources. Examples

POST apis/default => create default
DELTE apis/default => delete default
POST apis/remote => create remote
DELETE apis/remote => delete default

Maybe also add GETs to see what is already there (?)

Reply:
Yes, this service is definitely not restul yet :smile: 
https://github.com/jvalue/open-data-service/pull/189#discussion_r487400473 has to be discussed before refactoring